tinygrad/tensor.py

```diff
@@ -2447,6 +2447,34 @@ def div(self, x:Union[Tensor, ConstType], reverse=False, upcast=True) -> Tensor:
     if upcast: numerator, denominator = numerator.cast(least_upper_float(numerator.dtype)), denominator.cast(least_upper_float(denominator.dtype))
     return F.Div.apply(numerator, denominator)
 
+  def bitwise_and(self, x:Union[Tensor, ConstType], reverse=False) -> Tensor:
```

Comment:
do you wanna add an assert for self.dtype.is_int?

Reply:
yes. i have added that one.